packages/bot-config-utils/src/bot-config-utils.ts

```diff
@@ -185,57 +185,62 @@ export class ConfigChecker<ConfigType> {
         owner: owner,
         repo: repo,
         pull_number: prNumber,
-        per_page: 100,
+        per_page: 50, // Currently 30 is GitHub's default.
```

Comment:
I don't think it's concurrency. I think it's the number of items per page.

Reply:
That's reasonable, this reduces the concurrency to 50 from 100.

I think that using an async iterator is still a good call, vs., auto pagination, as it can help take a load off an API (_we've been moving away from using auto-pagination for client libraries in favor of async iterators._).

Reply:
Can you update the title and the description accordingly?